app/controllers/application_controller.rb

```diff
@@ -142,6 +143,19 @@ def current_user_redirect_path
     current_user.getting_started? ? getting_started_path : stream_path
   end
 
+  def gon_set_appconfig
+    gon.push({
+      appConfig: {
+        chat: {
+          enabled: AppConfig.chat.enabled?
+        },
+        settings: {
+          podname: AppConfig.settings.pod_name
+        }
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.